src/ppx/grfx/vk/vk_device.cpp

```diff
@@ -259,6 +276,8 @@ Result Device::ConfigureFeatures(const grfx::DeviceCreateInfo* pCreateInfo, VkPh
     features.shaderStorageImageMultisample        = foundFeatures.shaderStorageImageMultisample;
     features.samplerAnisotropy                    = foundFeatures.samplerAnisotropy;
 
+    mHasMultiView = pCreateInfo->multiView;
```

Comment:
I believe this should take into account whether extension is available (the check on line 207 above and foundMultiviewFeatures.multiview)